### DISCUSSION TITLE
Renovate doesn't recognize compose.yaml

### DISCUSSION THREAD

**Discussion by sebdanielsson** [renovatebot/renovate#15525] [Suggest an Idea]
### How are you running Renovate?

WhiteSource Renovate hosted app on github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

I just installed renovate-bot and it's not recognizing my compose.yaml files. Only docker-compose.yaml

https://github.com/compose-spec/compose-spec/blob/master/spec.md#compose-file

Example repo:
https://github.com/SebDanielsson/compose-transmission-wireguard

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

No reproduction, but I have linked to a public repo where it occurs

**Comment by rarkins** [comment]
If you want Renovate to support a new manager type or fileMatch pattern for an existing manager then file a feature request, not a bug report.

**Reply by viceice** [reply]
Before we need a minimal reproduction to check if it's a new manager or simply a new filename for docker compose files

**Reply by sebdanielsson** [reply]
Should be just a new filename. The following names are valid for docker compose files:

* docker-compose.yml
* docker-compose.yaml
* compose.yaml
* compose.yml

**Reply by viceice** [reply]
So feel free to open a PR to extend this

https://github.com/renovatebot/renovate/blob/dca3418bbdbf1024ac6807e591562b4042bc846d/lib/modules/manager/docker-compose/index.ts#L10

**Reply by viceice** [reply]
Probably `'(^|/)(?:docker-)?compose[^/]*\\.ya?ml$'` will do it

**Reply by sebdanielsson** [reply]
Not sure if I did everything correct but I've opened a PR. #15532
